### ISSUE TITLE
F-Droid OpenSSL error

### ISSUE THREAD

**Issue by orthecreedence** [turtl/tracker#276]
```
Login failed -- Error: An error in the openssl library: certificate verify failed
```

More openssl crap. I am hoping it's a simple configuration problem, but it might be that part of the build process isn't running and the cert chain isn't being bundled properly. Probably the second one, now that I think about it.

**Comment by orthecreedence** [comment]
Just a heads up: this should (hopefully) be fixed and will be released under the F-Droid Turtl build v0.7.2.5 (whenever the next F-Droid package update is).

**Comment by orthecreedence** [comment]
Closing because this seems to be resolved now.
